include/struct_defs/struct_02013A04_t.h

```diff
@@ -1,7 +1,7 @@
 #ifndef POKEPLATINUM_STRUCT_02013A04_T_H
 #define POKEPLATINUM_STRUCT_02013A04_T_H
 
-struct UnkStruct_02013A04_t {
+struct ResourceMetadata_t {
```

Comment:
Nitpicking on this one since it's short and a good example.

In an earlier file `include/struct_decls/struct_02013A04_decl.h`, we have this typedef:

```c
typedef struct ResourceMetadata_t ResourceMetadata;
```

The general style we're using for the gen4 projects is to merge these together and make the struct definition transparent. So, we'd instead have the single file with this:

```c
typedef struct ResourceMetadata {
    const void * unk_00;
    u32 unk_04;
} ResourceMetadata;
```

Then remove the includes for the previous file:

```c
#include "struct_decls/struct_02013A04_decl.h"
```

And, finally, delete the decl file.